src/spdx_tools/spdx3/parser/json/json_parser.py

```diff
@@ -0,0 +1,114 @@
+# SPDX-FileCopyrightText: 2022 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+import json
+from typing import Union
+
+from beartype.typing import Dict, List
+from semantic_version import Version
+
+from spdx_tools.spdx3.model import HashAlgorithm
+from spdx_tools.spdx3.parser.model_definition_dicts import (
+    CLASS_DICT,
+    DATETIME_PROPERTIES,
+    ENUM_DICT,
+    KEYS_WITH_URI_VALUES,
+    SPECIALLY_TREATED_KEYS,
+    VERSION_PROPERTIES,
+)
+from spdx_tools.spdx3.payload import Payload
+from spdx_tools.spdx.casing_tools import camel_case_to_snake_case
+from spdx_tools.spdx.datetime_conversions import datetime_from_str
+
+
+def parse_from_file(file_name: str) -> Payload:
+    with open(file_name) as file:
+        input_as_dict: Dict = json.load(file)
+
+    context = input_as_dict["@context"]
+    custom_context = {}
+    if isinstance(context, list):
+        for entry in context:
+            if isinstance(entry, dict):
+                custom_context.update(entry)
+
+    if "@graph" in input_as_dict:
+        payload = parse_list_of_elements(input_as_dict["@graph"], custom_context)
+    else:  # in case of only a single serialized element
+        input_as_dict.pop("@context")
+        payload = Payload()
+        payload.add_element(parse_element_object(input_as_dict, custom_context))
+
+    return payload
+
+
+def parse_list_of_elements(list_of_elements: List[Dict], custom_context: Dict):
+    payload = Payload()
+    for element_object in list_of_elements:
+        payload.add_element(parse_element_object(element_object, custom_context))
+    return payload
+
+
+def parse_element_object(element_object: Union[Dict, str], custom_context: Dict):
+    if isinstance(element_object, str):
```

Comment:
In general, JSON also supports numbers, booleans and null directly. Don't we need to handle them in similar fashion?